catalogue/terraform/stack/main.tf

```diff
@@ -42,6 +42,7 @@ module "catalogue-service-17092020" {
     items_api_key_stage  = "catalogue_api/items/stage/api_key"
 
     PRISMIC_ACCESS_TOKEN = "prismic-model/prod/access-token"
+    PRISMIC_BEARER_TOKEN = "prismic-model/graphql/prismic_bearer_token"
```

Comment:
this is the one non-trivial change – this is present in the deployed apps, not in Terraform

I added it as a no-op change, but I wonder if we should delete it instead? Presumably there's no GraphQL code left in the catalogue app to use this?